client/client/app/shared/projectContext/index.js

```diff
@@ -1380,9 +1380,7 @@ export default class projectContext {
                 this._viewport = null;
             }
         }
-        if (forceVariantsFilter || referenceDidChange || vcfFilesChanged) {
-            this.filter({asDefault: true, callbacks});
-        }
+        this.filter({asDefault: true, callbacks});
```

Comment:
We should not request variants if vcd files did not change or reference did not change